src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -408,7 +408,7 @@ private bool IsFileSource()
 
             if (IsV2LocalRepository(root))
             {
-                var pathResolver = new PackagePathResolver(root, useSideBySidePaths: true);
+                var pathResolver = new PackagePathResolver(sourceUri.AbsolutePath, useSideBySidePaths: true);
```

Comment:
@natemcmaster Is there a reason for `var root = sourceUri.LocalPath` instead of `var root = sourceUri.AbsolutePath` [above](https://github.com/NuGet/NuGet.Client/blob/9240fad570be5b2386ac6edddb3627fbd9fb4650/src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs#L402)

Reply:
I don't think there's a good reason. I'd imagine they're the exact same in pretty much every code path, otherwise we would've hit some problems.